ISISComputingGroup/EPICS-upgrade
# Ticket2787: Harmonise Galil to Other Motor IOCs

See https://github.com/ISISComputingGroup/IBEX/issues/2787
This will upgrade any Galil IOCs/Galil.cmds to be the new style.

The specific changes that should occur are:
* If a configuration contains a galil then the GALILADDRXX=IP macro should be split into MTRCTRL=XX and GALILADDR=IP
* All galilX.cmd files should be changed to contain the GALILADDR macro and their name changed to galil0X.cmd

To test:
* Confirm unit tests work
* Check the above changes occur when the upgrade script is run
* Check that no other changes occur in the configurations
